main/FatesRestartInterfaceMod.F90

```diff
@@ -927,13 +837,126 @@ subroutine define_restart_vars(this, initialize_variables)
          long_name='last 10 days of volumetric soil water, by site x day-index', &
          units='m3/m3', flushval = flushzero, &
          hlms='CLM:ALM', initialize=initialize_variables, ivar=ivar, index = ir_watermem_siwm )
-         
+
+
+    ! Register all of the PRT states and fluxes
+
+    ir_prt_base = ivar
+    call this%DefinePRTRestartVars(initialize_variables,ivar)
+       
+ 
     
     ! Must be last thing before return
     this%num_restart_vars_ = ivar
     
-  end subroutine define_restart_vars
-    
+ end subroutine define_restart_vars
+  
+  ! =====================================================================================
+  
+  subroutine DefinePRTRestartVars(this,initialize_variables,ivar)
+
```

Comment:
I'm not very sure what is going on in this subroutine. Maybe add some comment about what it is generally for.  I'm primarily worrying about the case of someone trying to add a new variable and not knowing how to make that happen here.